### ISSUE TITLE
[ACTIVE] SWARM-CLAIM D03|GUTENBERG-TERMINAL-SEAL|CURRENT-MAIN-CONVERGENCE|V1 — converge exact #627 authority

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1129]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_LANE_KEY: D03|GUTENBERG-TERMINAL-SEAL|CURRENT-MAIN-CONVERGENCE|V1
INSTANCE_TAG: SOL-GUTENBERG-SEAL-4N8Q
PREFERRED_WORK_KIND: CONVERGENCE / DATA / INTEGRATION_REPAIR
PARENT_ISSUES: #4 #548 #530 #592 #627 #1002
STATUS: ACTIVE_CLAIM_PENDING_POSTCLAIM_ARBITRATION
BASE_MAIN_AT_CLAIM: bcd7bc94713862ae5d4b40f50750db7bfe42d9be
CANONICAL_HISTORICAL_AUTHORITY: PR #627 exact head c50b3f9cf871792c03886bdc1ccdc144812be88f

WHY THIS PACKAGE:
- current main does not contain `en.project-gutenberg.public-domain-books` or the #627 terminal seal files;
- PR #627 already carries exact executed source evidence for three admitted normalized Gutenberg bodies: 1,672,110 UTF-8 bytes, exact content hashes, workflow run 32998859164 SUCCESS, and a conservative training-purpose-only rights boundary;
- #627 explicitly says its next action is source-registry/current-line convergence; rebuilding the Common Pile Gutenberg materializer (#1002/#877) would duplicate a weaker same-family path;
- central Rada/privacy/global-dedup/tokenizer/D05/D10 lanes are presently live-owned, so current-main convergence of this already-executed disjoint source authority is the highest-value collision-free finish-first package found.

PRECLAIM ARBITRATION:
- exact open search for family `en.project-gutenberg.public-domain-books` returned no issue owner;
- open search for `PR #627` / `PR627` / `Gutenberg terminal` returned no competing Gutenberg seal/current-main convergence owner; #1096 only references #627/#470 as the already-terminal reason not to rebuild #1002;
- historical #530 source-registry convergence predates SWARM-300-V2 and has no current lease; this claim does not rewrite the registry architecture;
- #1002 stale Common Pile Project Gutenberg materializer lane is a different and lower-authority implementation path; this claim consumes #627 exactly and does not mutate #1002.

CANDIDATE COMPARISON:
1. Exact #627 Gutenberg terminal-seal current-main convergence — SELECTED: already executed, exact-green source authority, no current owner, additive and immediately reusable by later global composition.
2. LoC #987 current-main repair — stale/unowned-looking but remains REVIEW_REQUIRED zero-credit and requires parser/product reconstruction; lower finish-first value.
3. USGPO #1001 stale materializer — requires fresh implementation/right review and ~208 MB transport; lower closure efficiency.
4. Central Rada/D04/D05/D10 repairs — higher leverage but actively owned; YIELD to incumbents.

OWNED SURFACE:
- transplant the exact four #627 authority files onto live current main without changing source bytes, normalized identities, rights decision, or source semantics;
- only minimal current-main compatibility fixes if exact tests prove necessary; any semantic change invalidates historical seal identity and must fail closed rather than silently widening authority;
- focused/adversarial LOCAL_FREE validation + exact-head shared CI;
- fresh independent audit handoff; producer does not self-certify or merge while #1045 integration ownership is live.

AVOID:
- no new Gutenberg materializer, no source acquisition rerun unless exact authority unexpectedly cannot be verified;
- no mutation of global dedup/decontamination/privacy/balance/split/packing/tokenizer/D05/D06/D10/model/trainer surfaces;
- no capacity promotion beyond terminal source admission; unique-loss/optimized-target authority remains 0;
- no final-test access, paid compute, external LLM/API data/intelligence, foreign pretrained weights, tokenizer fit, optimizer/model training.

ACCEPTANCE:
1. postclaim exact semantic arbitration still finds no earlier live owner;
2. current-main successor preserves exact #627 config authority identity `1b1bad11b688826ee4f73701c08e3b5af76ba16e8d8a806e008d5b84bee0b97b`, family identity `5798a9625d3b7f111de28851f7ef57d2a4f60d7b508987d24ce5905f0093fa12`, exact record hashes/byte counts and historical execution references;
3. validator/tests fail closed on authority/record/right/truth-boundary substitution;
4. source authority remains terminal-source-only: corpus/shard identity null, unique loss positions 0, no training execution;
5. exact-head shared CI terminal SUCCESS and fresh independent audit before integration.

TRUTH_BOUNDARY:
REAL_DATA_EXECUTED_BY_THIS_CLAIM=false (consumes historical #470/#627 real execution evidence only)
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
DURABLE PRODUCT CHECKPOINT — current-main Gutenberg terminal seal convergence

PR: #1130
Branch: `d03/gutenberg-terminal-seal-current-main-1129`
Base: `main@bcd7bc94713862ae5d4b40f50750db7bfe42d9be`
Exact Product head: `c2abacb2bcddc1b84b056ae55ed3108c3a9438fe`
Changed surface: exactly 4 additive files, 347 additions / 0 deletions.

Historical #627 blob equality independently re-read after publication:
- config `a4b2a2d20653fe91ea8520cd947e2f0bf54268b2` = exact #627 blob;
- validator `2d9714af31fca5f1a9fc1469f0e488387607382c` = exact #627 blob;
- tests `3114836b7df9d32203c585cc5306a235a5e587a0` = exact #627 blob;
- doc `a8d713475b565ffe8ebae1c1d54261f957201ae0` = exact #627 blob.

Current canonical shared CI: run `34553297083`, status `queued`; therefore NOT PASS and no integration claim is made yet.

Scientific boundary is unchanged from the sealed historical source authority: exact 3 normalized records / 1,672,110 UTF-8 bytes / one EN lineage family; evaluation NOT_AUTHORIZED; corpus/shard identities null; unique-loss and optimized-target authority 0. This claim executed no new source payload and no model training.

NEXT: fresh independent exact-head audit + terminal canonical CI. If both qualify this unchanged head, hand off to live integrator; any head movement invalidates the audit target.
